packages/stockflux-components/src/components/buttons/borderless-buttons/Close.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { FaTimes } from 'react-icons/fa';
+import cx from 'classnames';
+import BorderlessButton from '../borderless-button/BorderlessButton';
+
+export default props => (
+  <BorderlessButton
+    {...props}
+    className={cx('close', { [props.className]: props.className })}
+  >
+    <FaTimes />
```

Comment:
Which symbol is this using? If it closes a window should it be a cross symbol?

Reply:
The times symbol is used for close as it gives the normal "x" icon. the cross icon from FA gives the christian cross.
![image](https://user-images.githubusercontent.com/49903895/67856328-2f1b2c00-fb0c-11e9-8a5d-3efaf1a6001c.png)
When using FA to look for icons, the times icon is the one presented when looking for close as well. [https://fontawesome.com/icons?d=gallery&q=Close](https://fontawesome.com/icons?d=gallery&q=Close)